administration/src/util/helper.ts

```diff
@@ -1,8 +1,9 @@
 import XRegExp from 'xregexp'
 
-export const isProductionEnvironment = (): boolean => process.env.REACT_APP_IS_PRODUCTION === 'true'
-export const isDevMode = (): boolean => window.location.hostname === 'localhost'
-export const isStagingMode = (): boolean => !!window.location.hostname.match(/staging./)
+export const isStagingEnvironment = (): boolean => !!window.location.hostname.match(/staging./)
+export const isProductionEnvironment = (): boolean =>
+  process.env.REACT_APP_IS_PRODUCTION === 'true' && !isStagingEnvironment()
+export const isDevEnvironment = (): boolean => window.location.hostname === 'localhost'
```

Comment:
```suggestion
export const isDevEnvironment = (): boolean => process.env.REACT_APP_IS_PRODUCTION !== 'true'
```

Reply:
in the end it turned out that we don't need this helper function. i just adjusted some checks and fixed the tests